microsoft-edge/devtools-guide-chromium/whats-new/2021/05/devtools.md

```diff
@@ -128,29 +162,192 @@ Version 1.1.8 of the [Microsoft Edge Developer Tools for Visual Studio Code][Vis
 
-## Getting in touch with Microsoft Edge DevTools team  
+:::image type="complex" source="../../media/2021/05/css-grid-editor.msft.png" alt-text="CSS Grid editor" lightbox="../../media/2021/05/css-grid-editor.msft.png":::
+   CSS Grid editor
+:::image-end:::
+
+<!-- screenshot uses https://jec.fyi -->
+
+To review the history of this feature in the Chromium open-source project, navigate to Issue [1203241][CR1203241].
+
+
+### Support for const redeclarations in the Console
+
+The Console now supports redeclaration of `const` variables within a given REPL script (a series of statements entered as a group), in addition to the existing `let` and `class` redeclarations. The inability to redeclare `const` variables was a common annoyance for web developers who use the Console to experiment with new JavaScript code.
+
+This support for redeclaration of `const` variables allows yet to paste code into the DevTools console to see how it works.  You can experiment, making small changes to the code, and repeat the process without refreshing the page.  Previously, DevTools threw a syntax error if the code redeclared a `const` binding.
+
+Refer to the example below. `const` redeclaration is supported across separate REPL scripts (refer to variable `a`).  Note that the following scenarios are not supported, by design:
+
+*  `const` redeclaration of page scripts is not allowed in REPL scripts.
+*  `const` redeclaration within the same REPL script is not allowed (refer to variable `b`).
+
+:::image type="complex" source="../../media/2021/05/support-for-const-redeclaration.msft.png" alt-text="Redeclaring a const variable is allowed in the console" lightbox="../../media/2021/05/support-for-const-redeclaration.msft.png":::
+   Redeclaring a const variable is allowed in the console
+:::image-end:::
+
+<!-- screenshot uses https://googlechrome.github.io/samples/intersectionobserver but could use any page, or no page -->
+
+For example, to produce the results shown in the previous screenshot:
+
+1.  In DevTools version 92 or later, in the **Console**, enter the following statement and then select `Enter`.
+
+    ```javascript
+    const a = 'declare';
+    ```
+    
+1.  Enter the following statement and then select `Enter`.  This is a separate REPL script, independent from the above script, so no error message appears.
+
+    ```javascript
+    const a = 'redeclare';
+    ```
+    
+1.  Enter both of the following statements as a single REPL script.  To do this, either copy and paste the following code listing into the **Console**, or type the first statement followed by `Shift`+`Enter` and then type the second statement followed by `Enter`.  The const variable is redeclared within the same REPL script, so now an error message appears.
+
+    ```javascript
+    const b = 'declare';
+    const b = 'no redeclaration on same REPL script';
+    ```
```

Comment:
I'm not sure we need this whole bit. I prefer conciseness within individual announcements. We could always say something like:

```markdown
To learn how to run a single REPL script or a multi-line REPL script, navigate to [The Console as a JavaScript environment](https://docs.microsoft.com/microsoft-edge/devtools-guide-chromium/console/console-javascript).
```